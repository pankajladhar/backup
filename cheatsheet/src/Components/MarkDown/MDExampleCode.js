import React from 'react';
import PropTypes from 'prop-types';

const MDExampleCode = (props) => {
    return (
        <div className="MDExample__Code">
            <div className="MDExample__Title">Code</div>
            <div className="MDExample__Content">
                {props.children}
            </div>
        </div>
    )
}

MDExampleCode.propTypes = {
    children: PropTypes.node
}

export default MDExampleCode;
