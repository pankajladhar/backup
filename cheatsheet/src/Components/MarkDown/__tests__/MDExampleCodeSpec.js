import React from 'react';
import renderer from 'react-test-renderer';
import MDExampleCode from './../MDExampleCode';

describe('<MDExampleCode />', () => {
    it('should render properly', () =>{
        let tree = renderer.create(
            <MDExampleCode>
                <span>children</span>
            </MDExampleCode>
        ).toJSON();

        expect(tree).toMatchSnapshot()
    })
})