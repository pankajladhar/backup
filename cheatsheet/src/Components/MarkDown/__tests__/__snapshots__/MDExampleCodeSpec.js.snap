// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MDExampleCode /> should render properly 1`] = `
<div
  className="MDExample__Code"
>
  <div
    className="MDExample__Title"
  >
    Code
  </div>
  <div
    className="MDExample__Content"
  >
    <span>
      children
    </span>
  </div>
</div>
`;
